app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/triage/singledisposals/summary.scala.html

```diff
@@ -45,7 +45,7 @@
 @userKey = @{
     representativeType match {
         case Some(PersonalRepresentative) => ".personalRep"
-        case Some(PersonalRepresentativeInPeriodOfAdmin) => ".personalRep"
+        case Some(PersonalRepresentativeInPeriodOfAdmin) => ".personalRepInPeriodOfAdmin"
```

Comment:
does this break the title of the disposal date and completion date questions in the cya table? They don't have titles with `personalRepInPeriodOfAdmin` in them